io.github.smal82.LinuxMagnetDistro/io.github.smal82.LinuxMagnetDistro.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Linux Magnet Distro
+Comment=L'hub per ISO Linux. Scopri, scarica e vivi la libertà del software open source.
+Exec=linuxmagnetdistro
+Icon=io.github.smal82.LinuxMagnetDistro
+Terminal=false
+Type=Application
+Categories=Utility;
+StartupNotify=true
+# Optional
+Homepage=https://github.com/smal82/Linux-Magnet-Distro-GNOME
```

Comment:
I don't see where this is an existing key https://specifications.freedesktop.org/desktop-entry-spec/1.5/recognized-keys.html